web/modules/custom/joinup_core/joinup_core.post_update.php

```diff
@@ -74,3 +74,14 @@ function joinup_core_post_update_move_contact_form_attachments() {
 function joinup_core_post_update_install_smart_trim() {
   \Drupal::service('module_installer')->install(['smart_trim']);
 }
+
+/**
+ * Sets the PIWIK batch size to 50 items and item limit to 1000.
+ */
+function joinup_core_post_update_piwik_bulk_size() {
+  // This will allow to mainly always timeout, otherwise keep processing.
+  \Drupal::configFactory()->getEditable('cached_computed_field.settings')
```

Comment:
It is not needed to update config using a post-update hook, just update the exported config. It will be automatically synced on deployment.

Reply:
It wasn't working when I tried it... I removed it.